Pose start
Pose end
---
Pose[] path